docs/monitoring/logging.md

```diff
@@ -38,6 +38,23 @@ Flink ships with the following default properties files:
 - `log4j-yarn-session.properties`: Used by the Flink command line client when starting a YARN session (`yarn-session.sh`)
 - `log4j.properties`: JobManager/Taskmanager logs (both standalone and YARN)
 
+### Compatibility with Log4j1
+
+Flink ships with the [Log4j API bridge](https://logging.apache.org/log4j/log4j-2.2/log4j-1.2-api/index.html), allowing existing applications that work against Log4j1 classes to continue working.
+
+If you have custom Log4j1 properties files or code that relies on Log4j1, please check out the official Log4j [compatibility](https://logging.apache.org/log4j/2.x/manual/compatibility.html) and [migration](https://logging.apache.org/log4j/2.x/manual/migration.html) guides.
+
+## Configuring Log4j1
+
+To use Flink with Log4j1 you must ensure that:
+- `org.apache.logging.log4j:log4j-slf4j-impl` and `org.apache.logging.log4j:log4j-1.2-api` are not on the classpath,
+- `org.slf4j:slf4j-log4j12`and `log4j:log4j` are on the classpath.
+
+In the IDE this means to replacing such dependencies defined in your pom, and possible adding exclusions on dependencies that transitively depend on them.
+
+For Flink distributions this means to replace the respective jars in the `lib` directory, and
+replacing all log4j properties files in the `conf` directory with Log4j1-compliant versions.
+
```

Comment:
Maybe also a note that native log4j2 deps should stay in /lib for ES

Reply:
I'd prefer not to reference ES directly since such a statement is easily out-dated, or rather insufficient.
But I'll add a note that log4j-api must remain in /lib.